task006/task_02/client/client.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"bufio"   // buffers for reading
+	"context" // pkg that allows us to set timeout
+	"fmt"     // output into terminal
+	"log"     // logging pkg
+	"net"     // main pkg for networking
+	"os"
+	"time" // pkg for time constant
+)
+
+func main() {
+	// create a dialer
```

Comment:
Try not to leave comments in the code. Cases where comments need to be left are covered here https://golang.org/doc/effective_go#commentary

In good, self-documented code, you don't have to explain every single line because every identifier (variable, method, class) has a clear semantic name. Having more comments makes code hard to read